.circleci/config.yml

```diff
@@ -58,6 +58,9 @@ jobs:
       - image: circleci/rust:1.46.0
     steps:
       - checkout
+      - run:
+          name: Install linkchecker dependency
+          command: pip3 install --user git+https://github.com/linkchecker/linkchecker.git
```

Comment:
There's also https://github.com/deadlinks/cargo-deadlinks

Reply:
Aleksey, is there any rust crate you don't know about ?